### PR TITLE
Updated gdml to geometry as constructed for 2022 test beam

### PR DESCRIPTION
**Pull request by PeterGy**
Redefined detecting layers as: 0.5 mm Al cover, 20 mm scintillator, 3 mm steel holding plate. In this order.
Changed absorber into a plate without holes, with thickness 20 mm, 
Added 20th absorber
Implemented x bar displacement constant
Added 8020 frame

Added 2 LYSO options, currently commented
Added code that could be used for absorber holes

### CODE REVIEW

**Comment by EinarElen** [comment]
Hi, 

why have you deleted a bunch of files? e.g. the reference code and the figures?

**Review by EinarElen — CHANGES_REQUESTED** [review]
OK, I haven't had time to look at the documentation but here's my feedback.

Cheers!

**Comment by PeterGy** [comment]
Great feedback. I shall address your comments in the form of an excellent fix.

File: materials.gdml [hunk 1]
```diff
@@ -101,6 +101,23 @@
...
+	  <material name="Aluminum" state="solid">
+         <T unit="K" value="293.15"/>
+         <D unit="g/cm3" value="2.6989"/>
+         <fraction n="1" ref="Al"/>
+      </material>
+	  <material name="Aluminum3003" state="solid">
+         <T unit="K" value="293.15"/>
+         <D unit="g/cm3" value="2.6989"/>
+         <fraction n="0.99" ref="Al"/>
+         <fraction n="0.01" ref="Mn"/>
```
[the remark this answers is not in the document]
Comment (PeterGy) [reply materials.gdml:120]: I looked up the definition on wikipedia. They did say it varies very little bit, and this was the simplest way of matching the general criteria.

File: hcal_volumes.gdml [hunk 2]
```diff
@@ -5,6 +5,22 @@
   <auxiliary auxtype="VisAttributes" auxvalue="HcalVis"/>
 </volume>
 
+<volume name="vertical_bar_mounting_plate_volume">
+  <materialref ref="Steel"/>
+  <solidref    ref="vertical_bar_mounting_plate_box"/>
+  <auxiliary auxtype="Color" auxvalue="Red"/>
+  <auxiliary auxtype="VisAttributes" auxvalue="HcalVis"/>
+</volume>
+
```
[the remark this answers is not in the document]
Comment (PeterGy) [reply hcal_volumes.gdml:14]: Yes, they are both steel.

**Comment by PeterGy** [comment]
All issues fixed in new push.

File: userinfo.gdml [hunk 3]
```diff
@@ -1,40 +1,62 @@
-    <!-- detector version -->
-    <auxiliary auxtype="DetectorVersion" auxvalue="0">
-      <auxiliary auxtype="DetectorName" auxvalue="ldmx-hcal-prototype"/>
-      <auxiliary auxtype="Author" auxvalue="Ruth Poettgen"/>
-      <auxiliary auxtype="Description" auxvalue="A first attempt of an HCal prototype geometry."/>
-    </auxiliary>
+<!-- detector version -->
+<auxiliary auxtype="DetectorVersion" auxvalue="0">
+  <auxiliary auxtype="DetectorName" auxvalue="ldmx-hcal-prototype-v1.0"/>
+  <auxiliary auxtype="Author" auxvalue="Ruth Poettgen"/>
```
Comment (EinarElen) [inline userinfo.gdml:4]: I guess we should update this before actually putting things into ldmx-sw

**Review by EinarElen — APPROVED** [review]

**Pull request merged**